l10n_br_stock_account/models/stock_rule.py

```diff
@@ -43,8 +43,10 @@ def _get_stock_move_values(self, product_id, product_qty, product_uom,
             product_id, product_qty, product_uom, location_id, name, origin, values,
             group_id
         )
-        move_values['fiscal_operation_id'] = self.fiscal_operation_id.id
-        move_values['invoice_state'] = self.invoice_state
+        move_values['fiscal_operation_id'] = move_values.get(
```

Comment:
Isso esta errado porque o método get_stock_move_values não pode pegar o fiscal_operation_id nem o invoice_status da move_values, esse método é chamado quando é rodado o método run do procurement e se for criado um pedido de venda ao confirmar o pedido de venda, será executado esse método para a rota "make to order" e também vai ser rodado para as rotas manufaturado ou comprado, e ai neste caso essas movimetações estarão erradas.

Reply:
Alguma sugestão alternativa?